static/app/views/dashboardsV2/widgetCard/widgetQueries.tsx

```diff
@@ -321,6 +321,17 @@ class WidgetQueries extends React.Component<Props, State> {
           referrer: `api.dashboards.widget.${displayType}-chart`,
           partial: true,
         };
+
+        if (
+          organization.features.includes('new-widget-builder-experience-design') &&
+          [DisplayType.AREA, DisplayType.BAR, DisplayType.LINE].includes(displayType) &&
```

Comment:
do we need to have this check?

Reply:
I think so, because this portion handles everything that isn't a `TOP_N` display type. We only want to inject the `topEvents`, `orderBy` and `field` stuff when we're looking at a time series chart and there's at least one column to group by